README.md

```diff
@@ -1,5 +1,137 @@
-# DaiPOS
+# Sign In View
 
-[![Build Status](https://travis-ci.org/DaiPOS/daipos.svg?branch=master)](https://travis-ci.org/DaiPOS/daipos)
+## Description
 
-Easy, web-based point-of-sale for DAI and xDAI.
+This will present a "Sign In" page to the vendor allowing them to enter their
+username and password. A link to "Sign In" will be present in the menu for the
+present time. Upon successful sign in, "Signed in as <username>" will render at
+the top of "Inventory", "Sales Report", and "Sign In"
+
+**Start Date : 23/12/2018**
+
+**Expected Completion Date : 24/12/1984**
+
+**Actual Completion Date : [TBD]**
+
+## Post-Mortem
+
+**Post-Mortem TBD**
+
+- Mistakes
+  - Sign In and Sign Up components were already built
+  - TBD
+- ## Challenges
+  - Having trouble passing authentication status through store to header
+- ## Metrics
+- ## Learnings
+
+## Goals
+
+- Persist the data for a single vendor
+- First step to transferring money from DaiPOS account to 3rd party wallets
+
+## Metrics
+
+- Actions
+
+  - Added
+
+  - Changed
+
+  - Removed
+
+- Expectations
+  - More
+    - Vendor specific read/write operations to the database
+  - Less
+    - Data persisted through the browser cache
+
+## R&D
+
+- Problem: Currently the app does not recognize who the user is
+
+  - Describe results you obtained through research and development
+    - No research started
+  - Solution: TBD
+
+- Problem: Currently the app persists data to the browser cache
+  - Describe results you obtained through research and development
+    - No research started
+  - Solution: TBD
+
+## Flow
+
+1. This is the "when?" and "where?" of this feature
+
+- Vendor is prompted to sign in
+- Vendor enters data into the displayed fields
+- Sign in success is displayed
+- Vendor is redirected to the "Inventory" view
+- Vendor sees "Signed in as <username>" in the header
+
+## Sub-Features
+
+- [x] Sign In button is in menu
+- [x] Sign In button redirects vendor to Sign In page
+- [x] Unregistered vendors have the option to Sign Up at the Sign In page
+- [ ] Unregistered vendors are advised to Sign Up if their account has more
+      than 0 Dai
+- [ ] Sign In submit sends credentials to Firebase
+- [ ] Authentication status is stored in state
+- [ ] Successful response from Firebase alerts vendor of sign in success
+- [ ] Vendor is routed to "Inventory"
+- [ ] "Signed in as <username>" is displayed in the header
+- [ ] Once your architecture is merged, these must be moved to stories in Pivotal Tracker
+- [ ] Eg. `<ConversationsContainer>`: A container that will act as the root for the new `<Stack>`, left of `<NotificationsContainer>`
+
+## Migration (optional)
+
+- How will the current architecture be modified to migrate to this new architecture?
+- What files will be removed?
+- What functions will be deprecated?
+
+## API
```

Comment:
Fill in the api section. Think of all the files you'll need to add and have a section for every of them like so:

<pre>
### redux/actions/user.js
```js
/**
 * Attempts to sign in a user.
 */
async function signIn({ usernameOrEmail, password }) {}

/**
 * Attempts to sign up a new user.
 */
async function signUp({ username, email, password }) {}

/**
 * Sign up a new user. Signs user in if it already exists.
 */
async function signUpWithOauth({ username, email, password }) {}
```

### redux/actions/banner.js
```js
/**
 * Shows a banner
 */
function show({ text, color }) {}
```
</pre>
^ Function signatures and constants. This is all copy-pastable when it comes to actual coding. This way coding will be very easy.

Reply:
Modified the documentation to this effect.  I need to practice with it so I'll leave this as unresolved.